### PR TITLE
populate shedyamls for all devteams' tools

### PR DESCRIPTION
**Pull request by martenson**
Some of the tools don't have categories (which means they have no category in the MTS either). Please contribute if you can.

### CODE REVIEW

File: tools/blat_coverage_report/.shed.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/blat_coverage_report
-owner: devteam
+categories: []
```
Comment (nsoranzo) [inline tools/blat_coverage_report/.shed.yml]: `Next Gen Mappers` and `Sequence Analysis`

**Comment by davebx** [comment]
As long as travis is happy, I'm giving this every :+1: I can muster. Which is one.

File: tools/blat_mapping/.shed.yml [hunk 2]
```diff
@@ -1,4 +1,10 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/blat_mapping
-owner: devteam
+categories: []
```
Comment (nsoranzo) [inline tools/blat_mapping/.shed.yml]: `Next Gen Mappers` and `Sequence Analysis`

File: tools/bwa/.shed.yml [hunk 3]
```diff
@@ -1,4 +1,9 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/bwa
-owner: devteam
+categories:
+- Next Gen Mappers
+description: Wrapper for bwa mem. aln, sampe, and samse
```
Comment (nsoranzo) [inline tools/bwa/.shed.yml]: `s/mem\./mem/`

File: tools/canonical_correlation_analysis/.shed.yml [hunk 4]
```diff
@@ -1,4 +1,13 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/canonical_correlation_analysis
-owner: devteam
+categories: []
```
Comment (nsoranzo) [inline tools/canonical_correlation_analysis/.shed.yml]: `Statistics`

File: tools/clustalw/.shed.yml [hunk 5]
```diff
@@ -1,4 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/clustalw
-owner: devteam
+categories:
+- Fasta Manipulation
```
Comment (nsoranzo) [inline tools/clustalw/.shed.yml]: I'd suggest to remove `Fasta Manipulation` and to add `Phylogenetics`.

File: tools/canonical_correlation_analysis/.shed.yml [hunk 6]
```diff
@@ -1,4 +1,13 @@
...
+description: Canonical Correlation Analysis
+long_description: 'This tool uses functions from ''yacca'' library from R statistical
+  package to perform Canonical Correlation Analysis (CCA) on the input data. It outputs
+  two files, one containing the summary statistics of the performed CCA, and the other
+  containing helioplots, which display structural loadings of X and Y variables on
+  different canonical components.
+
```
Comment (davebx) [inline tools/canonical_correlation_analysis/.shed.yml]: My most trivial nitpick yet: these newlines are probably superfluous.

File: tools/convert_solid_color2nuc/.shed.yml [hunk 7]
```diff
@@ -1,4 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/convert_solid_color2nuc
-owner: devteam
+categories: []
```
Comment (davebx) [inline tools/convert_solid_color2nuc/.shed.yml]: Fasta Manipulation, maybe?

File: tools/compute_q_values/.shed.yml [hunk 8]
```diff
@@ -1,4 +1,12 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/compute_q_values
-owner: devteam
+categories:
+- Sequence Analysis
+- Variant Analysis
```
Comment (nsoranzo) [inline tools/compute_q_values/.shed.yml]: I'd remove both and add `Statistics`.

File: tools/convert_solid_color2nuc/.shed.yml [hunk 9]
```diff
@@ -1,4 +1,11 @@
...
+description: Convert Color Space to Nucleotides
+long_description: 'The tool was designed for color space files generated from an ABI
+  SOLiD sequencer. The file format must be fasta-like: the title starts with a ">"
+  character, and each color space sequence starts with a leading nucleotide.
+
```
Comment (davebx) [inline tools/convert_solid_color2nuc/.shed.yml]: Your script added some newlines here, too.

File: tools/fastqsolexa_to_fasta_qual/.shed.yml [hunk 10]
```diff
@@ -1,4 +1,9 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/fastqsolexa_to_fasta_qual
-owner: devteam
+categories: []
```
Comment (davebx) [inline tools/fastqsolexa_to_fasta_qual/.shed.yml]: I think we have a category called Fastq Manipulation, too

File: tools/convert_solid_color2nuc/.shed.yml (hunk 7 above)
Comment (nsoranzo) [inline tools/convert_solid_color2nuc/.shed.yml]: `Fasta Manipulation`?

File: tools/fastqsolexa_to_fasta_qual/.shed.yml (hunk 10 above)
Comment (bgruening) [inline tools/fastqsolexa_to_fasta_qual/.shed.yml]: `Convert Formats`

File: tools/kernel_canonical_correlation_analysis/.shed.yml [hunk 11]
```diff
@@ -1,4 +1,9 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/kernel_canonical_correlation_analysis
-owner: devteam
+categories: []
```
Comment (bgruening) [inline tools/kernel_canonical_correlation_analysis/.shed.yml]: `Statistics`

File: tools/kernel_principal_component_analysis/.shed.yml [hunk 12]
```diff
@@ -1,4 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/kernel_principal_component_analysis
-owner: devteam
+categories: []
```
Comment (bgruening) [inline tools/kernel_principal_component_analysis/.shed.yml]: `Statistics`

File: tools/kernel_canonical_correlation_analysis/.shed.yml (hunk 11 above)
Comment (davebx) [inline tools/kernel_canonical_correlation_analysis/.shed.yml]: Statistics?

File: tools/kernel_principal_component_analysis/.shed.yml (hunk 12 above)
Comment (davebx) [inline tools/kernel_principal_component_analysis/.shed.yml]: Same as KCCA, I think. Statistics.

File: tools/mapping_to_ucsc/.shed.yml [hunk 13]
```diff
@@ -1,4 +1,9 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/mapping_to_ucsc
-owner: devteam
+categories: []
```
Comment (bgruening) [inline tools/mapping_to_ucsc/.shed.yml]: `Convert Formats` maybe `Visualization`

File: tools/emboss_5/.shed.yml [hunk 14]
```diff
@@ -1,4 +1,7 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/emboss_5
-owner: devteam
+categories:
+- Sequence Analysis
```
Comment (nsoranzo) [inline tools/emboss_5/.shed.yml]: Add `Fasta Manipulation`.

File: tools/fastqsolexa_to_fasta_qual/.shed.yml (hunk 10 above)
Comment (nsoranzo) [inline tools/fastqsolexa_to_fasta_qual/.shed.yml]: `Fastq Manipulation`

File: tools/principal_component_analysis/.shed.yml [hunk 15]
```diff
@@ -1,4 +1,12 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/principal_component_analysis
-owner: devteam
+categories: []
+description: Principal Component Analysis
```
Comment (davebx) [inline tools/principal_component_analysis/.shed.yml]: Pretty sure this one also fits in Statistics

File: tools/megablast_wrapper/.shed.yml [hunk 16]
```diff
@@ -1,4 +1,9 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/megablast_wrapper
-owner: devteam
+categories: []
```
Comment (bgruening) [inline tools/megablast_wrapper/.shed.yml]: `Next Gen Mappers` & `Sequence Analysis`

File: tools/principal_component_analysis/.shed.yml [hunk 17]
```diff
@@ -1,4 +1,12 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/principal_component_analysis
-owner: devteam
+categories: []
```
Comment (davebx) [inline tools/principal_component_analysis/.shed.yml]: Pretty sure this one also fits in Statistics

File: tools/generate_pc_lda_matrix/.shed.yml [hunk 18]
```diff
@@ -1,4 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/generate_pc_lda_matrix
-owner: devteam
+categories:
+- Sequence Analysis
+description: Generate A Matrix for using PC and LDA
```
Comment (nsoranzo) [inline tools/generate_pc_lda_matrix/.shed.yml]: s/A Matrix/a matrix/

File: tools/kernel_canonical_correlation_analysis/.shed.yml (hunk 11 above)
Comment (nsoranzo) [inline tools/kernel_canonical_correlation_analysis/.shed.yml]: `Statistics`

File: tools/kernel_principal_component_analysis/.shed.yml (hunk 12 above)
Comment (nsoranzo) [inline tools/kernel_principal_component_analysis/.shed.yml]: `Statistics`

File: tools/linear_regression/.shed.yml [hunk 19]
```diff
@@ -1,4 +1,13 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/linear_regression
-owner: devteam
+categories:
+- Sequence Analysis
+- Variant Analysis
```
Comment (nsoranzo) [inline tools/linear_regression/.shed.yml]: Remove both and add `Statistics`.

File: tools/logistic_regression_vif/.shed.yml [hunk 20]
```diff
@@ -1,4 +1,13 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/logistic_regression_vif
-owner: devteam
+categories:
+- Sequence Analysis
+- Variant Analysis
```
Comment (nsoranzo) [inline tools/logistic_regression_vif/.shed.yml]: `Statistics`

File: tools/megablast_wrapper/.shed.yml (hunk 16 above)
Comment (nsoranzo) [inline tools/megablast_wrapper/.shed.yml]: `Next Gen Mappers`

File: tools/mapping_to_ucsc/.shed.yml (hunk 13 above)
Comment (nsoranzo) [inline tools/mapping_to_ucsc/.shed.yml]: `Next Gen Mappers`

File: tools/megablast_xml_parser/.shed.yml [hunk 21]
```diff
@@ -1,4 +1,9 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/megablast_xml_parser
-owner: devteam
+categories: []
```
Comment (nsoranzo) [inline tools/megablast_xml_parser/.shed.yml]: `Next Gen Mappers`

Comment (bgruening) [inline tools/megablast_xml_parser/.shed.yml]: `Convert Formats`

File: tools/partialr_square/.shed.yml [hunk 22]
```diff
@@ -1,4 +1,14 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/partialr_square
-owner: devteam
+categories:
+- Sequence Analysis
+- Variant Analysis
```
Comment (nsoranzo) [inline tools/partialr_square/.shed.yml]: `Statistics` instead.

File: tools/principal_component_analysis/.shed.yml (hunk 17 above)
Comment (bgruening) [inline tools/principal_component_analysis/.shed.yml]: `Statistics`

Comment (nsoranzo) [inline tools/principal_component_analysis/.shed.yml]: `Statistics`.

File: tools/rmap/.shed.yml [hunk 23]
```diff
@@ -1,4 +1,8 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/rmap
-owner: devteam
+categories: []
```
Comment (bgruening) [inline tools/rmap/.shed.yml]: `Next Gen Mappers`

Comment (nsoranzo) [inline tools/rmap/.shed.yml]: `Next Gen Mappers`.

File: tools/rmapq/.shed.yml [hunk 24]
```diff
@@ -1,4 +1,8 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/rmapq
-owner: devteam
+categories: []
```
Comment (nsoranzo) [inline tools/rmapq/.shed.yml]: `Next Gen Mappers`

Comment (bgruening) [inline tools/rmapq/.shed.yml]: `Next Gen Mappers`

File: tools/short_reads_figure_high_quality_length/.shed.yml [hunk 25]
```diff
@@ -1,4 +1,9 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/short_reads_figure_high_quality_length
-owner: devteam
+categories: []
```
Comment (nsoranzo) [inline tools/short_reads_figure_high_quality_length/.shed.yml]: `Sequence Analysis`

File: tools/short_reads_figure_score/.shed.yml [hunk 26]
```diff
@@ -1,4 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/short_reads_figure_score
-owner: devteam
+categories: []
```
Comment (nsoranzo) [inline tools/short_reads_figure_score/.shed.yml]: `Sequence Analysis`

File: tools/short_reads_trim_seq/.shed.yml [hunk 27]
```diff
@@ -1,4 +1,9 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/short_reads_trim_seq
-owner: devteam
+categories: []
```
Comment (nsoranzo) [inline tools/short_reads_trim_seq/.shed.yml]: `Fastq Manipulation`

File: tools/split_paired_reads/.shed.yml [hunk 28]
```diff
@@ -1,4 +1,11 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/split_paired_reads
-owner: devteam
+categories:
+- Fasta Manipulation
```
Comment (nsoranzo) [inline tools/split_paired_reads/.shed.yml]: s/Fasta/Fastq/

File: tools/short_reads_figure_high_quality_length/.shed.yml (hunk 25 above)
Comment (bgruening) [inline tools/short_reads_figure_high_quality_length/.shed.yml]: `Graphics` and we should have a category called `Quality Control`

File: tools/tophat/.shed.yml [hunk 29]
```diff
@@ -1,4 +1,13 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/tophat
-owner: devteam
+categories:
+- RNA
```
Comment (nsoranzo) [inline tools/tophat/.shed.yml]: Add `Next Gen Mappers`.

File: tools/tophat2/.shed.yml [hunk 30]
```diff
@@ -1,4 +1,13 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/tophat2
-owner: devteam
+categories:
+- RNA
```
Comment (nsoranzo) [inline tools/tophat2/.shed.yml]: Add `Next Gen Mappers`.

File: tools/short_reads_figure_score/.shed.yml (hunk 26 above)
Comment (bgruening) [inline tools/short_reads_figure_score/.shed.yml]: `Graphics` and we should have a category called `Quality Control`

File: tools/mapping_to_ucsc/.shed.yml (hunk 13 above)
Comment (davebx) [inline tools/mapping_to_ucsc/.shed.yml]: @nsoranzo Is it next gen, though?

File: tools/short_reads_trim_seq/.shed.yml (hunk 27 above)
Comment (bgruening) [inline tools/short_reads_trim_seq/.shed.yml]: `Fastq Manipulation`

File: tools/mapping_to_ucsc/.shed.yml (hunk 13 above)
Comment (nsoranzo) [inline tools/mapping_to_ucsc/.shed.yml]: This category would be better called Sequence Alignment, but that's what we have.

**Comment by martenson** [comment]
All above should be resolved; GitHub just does not know. Thanks guys.

**Comment by bgruening** [comment]
LGTM and is a huge improvement!
Thanks @martenson!

**Pull request merged**